src/main/java/org/example/UserGenerator.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class UserGenerator {
```

Comment:
This class does not have functionality. It keeps some data only and returns it.
We call such classes `Utility`.

A utility class is a class that is just a namespace for functions. No instances of it can exist, and all its members are static. For example, java.lang.Math and java.util.Collections are well known utility classes. This annotation automatically turns the annotated class into one.

A utility class cannot be instantiated. You need to generate a private constructor, mark the class final. If the class is an inner class, the class is also marked static.

All members of a utility class are static. Even fields and inner classes.

- [ ] private constructor
- [ ] final class
- [ ] static fields and methods